### PR TITLE
Add README.md

### PR DESCRIPTION
**Pull request by spbnick**

### CODE REVIEW

**Comment by spbnick** [comment]
@danghai, could you please review this?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  where `<N>` is the interface number (last number before `DESCRIPTOR` in
+  `usbhid-dump` output).
+
+See [Collecting tablet diagnostics][diagnostics_howto] HOWTO for instructions
+on collecting most of the above.
+
+Before submitting tablet information, please run `./validate` in the top
+directory to verify the data structure. You will need to have `kwalify`
+installed for it to work.
+
```
Comment (danghai) [inline README.md]: Do you think it is good to mention about running test local `digimend` website by `digimend.github.io` repo before submitting?

**Comment by danghai** [comment]
It is great!

**Comment by spbnick** [comment]
Thank you, Hai, merged!

**Pull request merged**